frontend/.eslintrc.js

```diff
@@ -3,116 +3,5 @@ module.exports = {
-      tsx: true,
-    },
-    ecmaVersion: 'latest',
-    sourceType: 'module',
-  },
-  plugins: ['react', 'react-hooks'],
-  rules: {
-    semi: ['error', 'always'],
-    quotes: ['error', 'single'],
-    'no-use-before-define': 0,
-    'no-await-in-loop': 0,
-    'no-continue': 0,
-    'no-plusplus': 0,
-    'no-var': 'error',
-    'no-restricted-syntax': ['warn'],
-    'no-extra-boolean-cast': 0,
-    'class-methods-use-this': 0,
-    'consistent-return': 0,
-    'no-return-assign': ['error', 'except-parens'],
-    'lines-between-class-members': 0,
-    'import/prefer-default-export': 0,
-    '@typescript-eslint/no-for-in-array': ['error'],
-    'no-unused-vars': ['warn', { args: 'none' }],
-    'no-useless-constructor': 0,
-    'no-empty-function': 0,
-    'no-unused-expressions': 'off',
-    'no-param-reassign': ['error', { props: false }],
-    'react/function-component-definition': 0,
-    'jsx-a11y/label-has-associated-control': 0,
-    '@typescript-eslint/no-unused-expressions': ['error'],
-    '@typescript-eslint/no-unsafe-argument': ['warn'],
-    '@typescript-eslint/no-unused-vars': ['warn'],
-    '@typescript-eslint/no-unsafe-member-access': ['warn'],
-    '@typescript-eslint/no-use-before-define': ['off'],
-    '@typescript-eslint/require-await': 'off',
-    '@typescript-eslint/prefer-optional-chain': ['warn'],
-    '@typescript-eslint/prefer-nullish-coalescing': 'off',
-    '@typescript-eslint/no-floating-promises': 'off',
-    '@typescript-eslint/no-unsafe-assignment': 'warn',
-    '@typescript-eslint/no-unsafe-call': 'warn',
-    '@typescript-eslint/no-unsafe-return': 'warn',
-    '@typescript-eslint/no-misused-promises': [
-      'error',
-      {
-        checksVoidReturn: false,
-      },
-    ],
-    'import/extensions': [
-      'error',
-      {
-        js: 'never',
-        jsx: 'never',
-        ts: 'never',
-        tsx: 'never',
-      },
-    ],
-    'import/no-extraneous-dependencies': [
-      'error',
-      {
-        devDependencies: true,
-      },
-    ],
-    'react/prop-types': 0,
-    'react/require-default-props': 0,
-    'react/jsx-filename-extension': [1, { extensions: ['.jsx', 'tsx'] }],
-    'react/jsx-closing-bracket-location': 0,
-    'react/jsx-props-no-spreading': 0,
-    'react/jsx-curly-newline': 0,
-    'react/jsx-curly-brace-presence': 0,
-    'react/no-array-index-key': 1,
-    'react/jsx-one-expression-per-line': 0,
-    'react/jsx-wrap-multilines': 0,
-    'react/jsx-indent': 0,
-  },
-  overrides: [
-    {
-      files: ['*.js', '*.jsx', '*.ts', '*.tsx'],
-      parserOptions: {
-        project: ['./tsconfig.json'],
-      },
-    },
-  ],
-  ignorePatterns: [
-    'yarn.lock',
-    '.eslintrc.js',
-    'tailwind.config.js',
-    './cypress/*',
-  ],
-  settings: {
-    'import/resolver': {
-      node: {
-        moduleDirectory: ['node_modules', 'src/'],
-      },
-    },
-    react: {
-      version: 'detect',
-    },
-  },
+  extends: ['react-app', 'react-app/jest', 'prettier'],
```

Comment:
There is also a plan in the future to create our own package to keep eslint configurations consistent project to project.  Let's not change this completely.  If you want to add or change a few rules let's chat about that.